tool/planet/constants.go

```diff
@@ -422,6 +425,10 @@ const (
 
 	// DefaultServiceNodePortRange defines the default IP range for services with NodePort visibility
 	DefaultServiceNodePortRange = "30000-32767"
+
+	// DefaultCriticalNamespaces is the default list of critical namespaces
```

Comment:
Critical w/o a context carries a different connotation - `... the default list of namespaces to monitor for pod health` or something along the lines.